app/views/stats/index.html.slim

```diff
@@ -5,4 +5,4 @@
     small Du #{l(Stat::DEFAULT_RANGE.first.to_date)} au #{l(Stat::DEFAULT_RANGE.last.to_date)}
 
 .container
-  = render 'statistics'
+  = render 'statistics', stats: @stats, rdvs_stats_path: rdvs_stats_path, users_stats_path: users_stats_path, service_stats_path: rdvs_stats_path(by_service: true)
```

Comment:
@yaf what I meant was : 
- it's a good idea to stop passing down the url helpers (rdvs_stats_path, users_stats_path) etc, as they are already available for use in the partials just like any other helpers
- it's not a good idea to stop explicitly passing down view instance variable like @stats. partials should not (in my opinion) access instance variables, but only local variables.